src/Microsoft.DotNet.Interactive.Jupyter.Tests/CompleteRequestHandlerTests.cs

```diff
@@ -33,5 +36,31 @@ public async Task send_completeReply_on_CompleteRequest()
                 .Should()
                 .ContainSingle(r => r is CompleteReply);
         }
+
+        [Fact]
+        public void cell_language_can_be_pulled_from_metadata_when_present()
+        {
+            var metaData = new Dictionary<string, object>()
+            {
+                { "dotnet_interactive", JObject.Parse(JsonConvert.SerializeObject(new { language = "fsharp" })) }
```

Comment:
Can we do this with the appropriate message type rather than a `JObject`?

Reply:
It comes back from the parser as `JObject` by default, and given that we may change this in the future, we'll be the most flexible if we stay this way.  @colombod, what's your take on it?

Reply:
This is part of why it stood out for me. Since `JObject` is an implementation detail and we plan to move to `System.Text.Json`, all the more reason to keep serializer types out of the tests.

Reply:
then metadata is definitely a Dictionary<string,object> and the values can be as deep as necessary.

Reply:
something like
![image](https://user-images.githubusercontent.com/375556/104730241-97febc00-5731-11eb-9777-bfa2720ff137.png)
![image](https://user-images.githubusercontent.com/375556/104730303-b369c700-5731-11eb-9c2d-c9ed80071123.png)

Reply:
done